src/ifcparse/IfcSpfHeader.cpp

```diff
@@ -17,161 +17,167 @@
  *                                                                              *
  ********************************************************************************/
 
-#include "../ifcparse/IfcFile.h"
-
 #include "IfcSpfHeader.h"
 
-static const char * const ISO_10303_21		= "ISO-10303-21";
-static const char * const HEADER			= "HEADER";
-static const char * const FILE_DESCRIPTION	= "FILE_DESCRIPTION";
-static const char * const FILE_NAME			= "FILE_NAME";
-static const char * const FILE_SCHEMA		= "FILE_SCHEMA";
-static const char * const ENDSEC			= "ENDSEC";
-static const char * const DATA				= "DATA";
+#include "IfcFile.h"
+
+static const char* const ISO_10303_21 = "ISO-10303-21";
+static const char* const HEADER = "HEADER";
+static const char* const FILE_DESCRIPTION = "FILE_DESCRIPTION";
+static const char* const FILE_NAME = "FILE_NAME";
+static const char* const FILE_SCHEMA = "FILE_SCHEMA";
+static const char* const ENDSEC = "ENDSEC";
+static const char* const DATA = "DATA";
 // The following header entities are not normally encountered in IFC files and are not parsed.
```

Comment:
needs to tweak `AlignConsecutiveAssignments` and its options

Reply:
Again, not a big issue for me. You already see in the previous state it got mangled between tab width and spaces. Besides https://clang.llvm.org/docs/ClangFormatStyleOptions.html#alignconsecutiveassignments does a lot of things I wouldn't consider to improve readability. Only in these cases where you're basically defining aliases it makes some sense to me, not in the general case of assignments. But I don't think clang-format will be able to meaningfully differentiate between those cases.

Reply:
Ok, we can leave this out for the start.